swingset/CHANGELOG.txt

```diff
@@ -3,13 +3,34 @@ ChangeLog file for the SwingSet Open Toolkit for Java Swing.
 ==============================================================================
 
 ==============================================================================
-SwingSet 4.0.11 - Released TBD
-==============================================================================
+SwingSet 4.0.11 - Released 2023-02-08
+==============================================================================
+
+	1. Due to the many version changes on an ongoing basis, starting with
+	   SwingSet 4.0.11, POM changes have been moved to CHANGELOG-POMS.txt
+	   in the same folder.
+	2. Fix for component updates following navigation:
+	   https://github.com/bpangburn/swingset/issues/144
+	3. SSComboBox/SSDBComboBox - Ability to display underlying database
+	   value when a combobox doesn't have a corresponding Option:
+	   https://github.com/bpangburn/swingset/issues/145
+	4. Add updateSSDBComboBoxes to SSDataGridScreenHelper:
+	   https://github.com/bpangburn/swingset/issues/153
+	5. Fix for SSTableModel draw after a deletion:
+	   https://github.com/bpangburn/swingset/issues/156
+	6. Fix for updating of default parent ID when using
+	   SSDataGridScreenHelper:
+	   https://github.com/bpangburn/swingset/issues/157
+	7. Addition of screen helper method to combine setParentID()
+	   and updateScreen():
+	   https://github.com/bpangburn/swingset/issues/159
+	8. Addition of Example4 Advanced to SwingSet demo to demonstrate
+	   #5 above.
```

Comment:
Actually, (I'll leave the wordsmithing to you)
```
#. Add Example4 Advanced. Moved code from Example4 that demos
   navigator actions: assigning actions to function keys, 
   assigning actions to buttons.

   Add demo code showing use of MissingOptionControl and
   custom ListItemFormat for null;/missing Options.

#. Add DataGridExampleSupport used by Example5.
   Provides more testing/examples of grid usage.
   Improved H2 handling so demo can now do row
   insertion/deletion into grid. Exposed 5 above.
```

Reply:
Edited CHANGELOG to pickup your changes. Thanks. Hopefully good enough. Full speed ahead.